spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -32,8 +32,9 @@
     end
 
     it 'returns the file even if it’s just ‘.reek’' do
-      found = described_class.find(current: CONFIG_PATH)
-      expect(found).to eq(CONFIG_PATH.join('.reek'))
+      single_configuration_file_dir = CONFIG_PATH.join('single_configuration_file')
+      found = described_class.find(current: single_configuration_file_dir)
+      expect(found).to eq(single_configuration_file_dir.join('.reek'))
```

Comment:
Maybe you should also add a specific spec for the case of two configuration files found? I think this would force you to not exit immediately but instead throw an error, but that would probably be a good thing.